examples/BlobUtilTester/testHelpers.ts

```diff
@@ -0,0 +1,30 @@
+export type TestResult = {
+  isSuccess: boolean;
+  message?: string;
+};
+
+export type TestFunction = () => Promise<TestResult>;
+
+export function assertEquals<T>(expected: T, actual: T): TestResult {
+  const isSuccess = expected === actual;
+
+  return {
+    isSuccess,
+    message: isSuccess
+      ? undefined
+      : `Expected "${expected}", but got "${actual}"`,
+  };
+}
```

Comment:
I think you could probably remove this in favor of importing the `expect` package?